<launch>
	<arg name="namespace" default="bow"/>
	<arg name="object" default="circle"/>
	<arg name="color" default="red"/>
	<arg name="debug" default="true"/>

	<group ns="$(arg namespace)">
		<node
			pkg="robotx_vision"
			type="depth_detection"
			name="depth_detection"
			required="true"
			output="screen"
		>
		<param name="subscribed_image_topic" value="rgb/image_rect_color"/>
		<param name="subscribed_depth_topic" value="depth/image_rect_color"/>
		<param name="object_shape" value="$(arg object)"/>
		<param name="object_color" value="$(arg color)"/>
		<param name="published_roi_topic" value="$(arg object)/$(arg color)/roi"/>
		<param name="published_distance_topic" value="$(arg object)/$(arg color)/distance"/>
		<param name="debug" value="$(arg debug)"/>

		</node>
	</group>
</launch>
